infra/aws/terraform/modules/eks-prow-iam/eks_resources_boundary.tf

```diff
@@ -54,7 +55,7 @@ data "aws_iam_policy_document" "provisioner_permission_boundary_doc" {
     ]
 
     resources = [
-      "arn:aws:iam::${local.account_id}:policy/TerraformPermissionBoundary"
+      "arn:aws:iam::${local.account_id}:policy/EKSResourcesPermissionBoundary"
```

Comment:
Should we also deny changing EKSInfraAdminPermissionBoundary?

Reply:
I like the idea

Reply:
If we use a path (see review comments elsewhere), we can deny changing       `arn:aws:iam::${local.account_id}:policy/boundary/*` (use the `ArnLike` operator in a Condition).